static/js/github-create-branch.js

```diff
@@ -86,16 +86,16 @@ const loadBranches = () => {
       const allBranchesfetched = data?.repository?.refs?.totalCount === data?.repository?.refs?.edges.length;
```

Comment:
Need to refactor this as well? can we return branches count?

Reply:
Getting the total number of branches is not possible with REST. There is no such property.

Reply:
Added another Request to fetch the default branch for the selected Repo